chandra_aca/planets.py

```diff
@@ -237,16 +236,17 @@ def get_planet_eci(
         Earth-Centered Inertial (ECI) position (km) as (x, y, z)
         or N x (x, y, z)
     """
-    time = CxoTime(time)
+    time_sec = convert_time_format(time, "secs")
 
-    pos_planet = get_planet_barycentric(body, time)
+    pos_planet = get_planet_barycentric(body, time_sec)
     if pos_observer is None:
         pos_observer = get_planet_barycentric("earth", time)
 
-    dist = np.sqrt(np.sum((pos_planet - pos_observer) ** 2, axis=-1)) * u.km
-    light_travel_time = (dist / const.c).to(u.s)
+    dist = np.sqrt(np.sum((pos_planet - pos_observer) ** 2, axis=-1))  # km
+    # Divide distance by the speed of light in km/s
+    light_travel_time = dist / 299792.458  # s
```

Comment:
And of course wondering if this should instead import the astropy constant and convert it to a unit-less number *once*.  But given that the speed of light is unlikely to change and this looks correct in km/s this is fine.